lib/sparse/src/index/inverted_index/inverted_index_compressed_immutable_ram.rs

```diff
@@ -98,16 +100,24 @@ impl<W: Weight> InvertedIndex for InvertedIndexCompressedImmutableRam<W> {
             }
             postings.push(new_posting_list.build());
         }
+
+        let total_sparse_size = postings.iter().map(|p| p.view().store_size().total).sum();
```

Comment:
Are we sure this is not slow?
Otherwise we moved the problem from storage to index :)